apps/vscode/command_client/command_client.py

```diff
@@ -197,20 +198,30 @@ def run_command(
     return decoded_contents["returnValue"]
 
 
+command_server_directory = None
+
+
+# NB: See https://github.com/talonhub/community/issues/966 for why we do OS-specific temp dirs
+def get_platform_specific_communication_dir_path():
+    home_dir = Path(os.path.expanduser("~"))
```

Comment:
`Path.home()` is a shorter way to spell this.

Reply:
Can anyone with permissions fix this before i test on neovim/Windows?

Reply:
done